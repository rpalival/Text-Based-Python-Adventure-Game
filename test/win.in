go north
get 3d maneuver gear
go east
get blade
go west
go south
go east
go north
go east
get light crystal
go north
inventory
go north