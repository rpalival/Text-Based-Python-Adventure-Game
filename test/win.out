> Wall Maria Entrance

The colossal gate of Wall Maria, towering above you. The air is tense with the threat of Titans.

Items: flare gun

Exits: north east

What would you like to do? You go north.

> Scout Regiment HQ

The bustling headquarters of the Scout Regiment. Maps and plans of Titan territory cover the walls.

Items: 3d maneuver gear

Exits: south east

What would you like to do? You pick up the 3d maneuver gear.
What would you like to do? You go east.

> Trost District Armory

A stockpile of weapons and supplies. The walls are adorned with swords and military insignia.

Items: blade

Exits: west south

What would you like to do? You pick up the blade.
What would you like to do? You go west.

> Scout Regiment HQ

The bustling headquarters of the Scout Regiment. Maps and plans of Titan territory cover the walls.

Exits: south east

What would you like to do? You go south.

> Wall Maria Entrance

The colossal gate of Wall Maria, towering above you. The air is tense with the threat of Titans.

Items: flare gun

Exits: north east

What would you like to do? You go east.

> Shiganshina District

The ruined district where it all began. Abandoned houses and the remnants of a battle linger.

Exits: north west

What would you like to do? You go north.

> Titan Forest

A dense forest, known for its massive trees. A perfect place for ambushes... or to be ambushed.

Exits: east south

What would you like to do? You go east.

> Stohess District Underground

A labyrinth of underground tunnels beneath Stohess District. It's easy to get lost in the darkness.

Items: light crystal

Exits: west north

What would you like to do? You pick up the light crystal.
What would you like to do? You go north.

> Survey Corps Expedition Start

The starting point for many expeditions beyond the walls. The air is filled with a mix of hope and fear.

Exits: west south north

What would you like to do? Inventory:
  3d maneuver gear
  blade
  light crystal
What would you like to do? You go north.

> Beast Titan's Lair

The ominous lair of the Beast Titan. The walls echo with the sounds of past battles and whispered secrets.

Exits: north

Congratulations! You have won the game!

